gemrb/tests/resources/INIImporter/sample.ini

```diff
@@ -2,7 +2,7 @@
 stringValue=abc
 ;oldValue=def
 intValue=123
-intValue=0
+intValue=1
```

Comment:
Rereading now how we parse, should we add another test and fix section detection? Anything without an equals sign is currently treated as one, not requiring being in brackets.

Reply:
Can you give me an example what you mean? I don't get it.

Reply:
```
[notASection
notASection]
notASection
```

Reply:
The `garbage` below doesn't for fall this. But I'll check the other stuff.

Reply:
Ah, I see, it's handled outside AddLine.